CONTRIBUTING.rst

```diff
@@ -0,0 +1,201 @@
+How To Contribute
+=================
+
+First off, thank you for considering contributing to ``attrs``!
```

Comment:
This should probably refer to `Numismatic`

Reply:
Yes, if you look closely, you may even find more mistakes. I will fix this all up on Sat (Sat I am devoting my whole day to this). However, we should chat quickly first. I need help on three things from you:
 1. The roadmap - I would like to give you my input to this, but I think since you are the lead on this project, you should... um, take the lead :) So lets chat about this
 2. The CLI - it would really help if you did this.
 3. Changelog - you are in a unique position to make this coherent.

Everything else I can do, from the philosophy of the project, to the explaining of the architecture, and the examples of extending. Of course, you will be able to check all this and make edits.